src/test/java/uk/gov/hmcts/probate/service/CaveatNotificationServiceTest.java

```diff
@@ -0,0 +1,222 @@
+package uk.gov.hmcts.probate.service;
```

Comment:
This test class is quite big - it may indicate that we can do a little bit of refactoring on the class itself - can you investigate please?

Reply:
This was the test class missing from master. Although we can refactor, this is already in master - do we want to do this under this pr or raise a new jira for refactoring?

Reply:
OK - keep this for now. We'll have to add to refactoring backlog